server/sonar-db-dao/src/main/java/org/sonar/db/organization/OrganizationDao.java

```diff
@@ -113,6 +113,10 @@ public void setDefaultGroupId(DbSession dbSession, String uuid, GroupDto default
     getMapper(dbSession).updateDefaultGroupId(uuid, requireNonNull(defaultGroupId, "Default group id cannot be null"), system2.now());
```

Comment:
ok, the case for WS api/projects/create is covered by SONAR-9091, so it all comes down to a wrong sentence : "first analysis" either means the first scanner execution for a project or the processing of the an analysis for a project for the first time. api/ce/submit is only upload of a report, not an analysis by itself